pep8.py

```diff
@@ -862,8 +887,8 @@ def module_imports_on_top_of_file(
 
     Okay: import os
     Okay: # this is a comment\nimport os
-    Okay: '''this is a module docstring'''\nimport os
-    Okay: r'''this is a module docstring'''\nimport os
+    Okay: ""\"this is a module docstring""\"\nimport os
```

Comment:
The trick is to have nested triple quoted strings. Not quite sure what is the best way to do so. Escaping every single double quote character brings a lot of visual noise. Escaping one was enough. Not quite sure what the favourite way to do it is.